packages/terra-tabs/src/StructuralTabs.scss

```diff
@@ -0,0 +1,130 @@
+// stylelint-disable selector-max-compound-selectors
+// stylelint-disable max-nesting-depth
+:local {
+  .structural {
+    background-color: var(--terra-tabs-structural-container-background-color, #fff);
+
+    // Content div of content container component
+    > :nth-child(2) {
+      background-color: var(--terra-tabs-structural-content-background-color, #fff);
+      border-radius: var(--terra-tabs-structural-content-border-radius, 0);
+    }
+
+    .collapsed-tabs-container {
+      border-bottom-color: var(--terra-tabs-structural-collapsed-menu-bar-border-color, #dddedf);
+      border-bottom-style: solid;
+      border-bottom-width: 1px;
+      border-radius: var(--terra-tabs-structural-collapsed-menu-bar-border-radius, 0);
+
+      .tab-menu:focus {
+        outline: none;
+      }
+
+      .tab-menu.is-active {
+        background-size: 0 0;
+      }
+
+      .tab-menu:hover {
+        background-size: 0 0;
+      }
+    }
+
+    .collapsible-tabs-container {
+      border-bottom-color: var(--terra-tabs-structural-tab-bar-border-color, #dddedf);
+      border-bottom-style: solid;
+      border-bottom-width: var(--terra-tabs-structural-tab-bar-border-width, 1px);
+    }
+
+    //Extended Tabs
+    &.tab-fill .collapsible-tabs-container:not(.is-calculating) {
+      .tab,
+      .tab-menu {
+        flex: 1 1 auto;
+      }
+    }
+
+    .tab,
+    .tab-menu {
+      background-color: var(--terra-tabs-structural-background-color, #fff);
+      background-image: var(--terra-tabs-structural-background-image, linear-gradient(to bottom, #007ee0, #007ee0));
+      background-position: var(--terra-tabs-structural-background-position, 0 100%);
+      background-repeat: no-repeat;
+      background-size: var(--terra-tabs-structural-background-size, 0 2px);
+      border-color: var(--terra-tabs-structural-border-color, #dedfe0);
+      border-radius: var(--terra-tabs-structural-border-radius, 0);
+      border-style: solid;
+      border-width: var(--terra-tabs-structural-border-width, 0 1px 0 0);
+      color: var(--terra-tabs-structural-color, #007ee0);
+      margin-left: var(--terra-tabs-structural-margin-between-tabs, 0);
+      transition-duration: var(--terra-tabs-structural-transition-duration, 0);
+      transition-property: background-size;
+      transition-timing-function: var(--terra-tabs-structural-transition-timing, ease);
+
+      &.is-active {
+        background-color: var(--terra-tabs-structural-active-background-color, #fff);
+        background-size: var(--terra-tabs-structural-active-background-size, 100% 2px);
+        color: var(--terra-tabs-structural-active-color, #000);
+      }
+
+      &.is-disabled,
+      &.is-disabled:hover {
+        background-color: var(--terra-tabs-structural-disabled-background-color, #fff);
+        background-size: 0 0;
+        color: var(--terra-tabs-structural-disabled-color, #909496);
```

Comment:
Lets update color from to `#909496` to `rgba(0,0,0,.25)` to better match inactive color used in other components